src/Nethermind/Nethermind.Consensus.Clique/CliquePlugin.cs

```diff
@@ -151,6 +151,11 @@ public Task<IBlockProducer> InitBlockProducer(IBlockProductionTrigger? blockProd
                 _cliqueConfig!,
                 getFromApi.LogManager);
 
+            if (blockProducer is IDisposable disposable)
```

Comment:
Change `blockProducer` variable type to be `CliqueBlockProducer`, so this check is not needed.

Reply:
changed